apps/src/templates/Notification.jsx

```diff
@@ -139,14 +143,18 @@ const Notification = React.createClass({
       bullhorn: 'bullhorn'
     };
 
+    const buttonStyle = type !== "course" ? styles.button : [styles.button, styles.courseButton];
```

Comment:
Also I think we could/should use `NotificationType.course` instead of `course` here and elsewhere in this file.